### DISCUSSION TITLE
Multiple AIO instances on one docker server

### DISCUSSION THREAD

**Discussion by MarcS1975** [nextcloud/all-in-one#5447] [Questions] [answered]
I have read the proposed description for running multiple instances of AIO [here ](https://github.com/nextcloud/all-in-one/blob/main/multiple-instances.md) but both methods seem very complicated, especially the root less docker variant.
I do not run Proxmox or other Hyperviser so I only have one instance of docker. Running one AIO instance behind Synology reverse proxy works fine and I thought that by simply amending some of the AIO container-names and respective port numbers in the docker compose file, I would be able to launch a second AIO inctsance, like I have done for many other applications, but this does not work.
Can someone explain why this is a problem and why we need this complicated rootless docker approach?
Name changes and port changes should isolate the data flow sufficiently for Docker . No?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/359#discussioncomment-2709650 and the whole discussion
